tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/management/BridgeControlTest.java

```diff
@@ -139,7 +138,7 @@ public void setUp() throws Exception {
 
       Configuration conf_0 = createBasicConfig().addAcceptorConfiguration(new TransportConfiguration(InVMAcceptorFactory.class.getName())).addConnectorConfiguration(connectorConfig.getName(), connectorConfig).addQueueConfiguration(sourceQueueConfig).addBridgeConfiguration(bridgeConfig);
 
-      server_1 = addServer(ActiveMQServers.newActiveMQServer(conf_1, MBeanServerFactory.createMBeanServer(), false));
+      server_1 = addServer(ActiveMQServers.newActiveMQServer(conf_1, getMBeanServer(), false));
```

Comment:
This would now be using the same MBeanServer as the other call below using the variable, mbeanServer, coming from the parent, rather than using two differnet MBeanServers as the different variables suggests...so you might as well make the two consistently use 1 if its acceptable for them to actually be using the same MBeanServer.